pages/views/components/esl-alert.njk

```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: ESL Alert
+description: Custom element to show small notifications on the page.
```

Comment:
That's exactly the title content. And here as it seems to me should be a brief description of what the page contains. I.e. something like this:

An explanation of the ESL Alert web component and its possible uses, as well as examples of how to use it.